distribution/Dockerfile

```diff
@@ -16,20 +16,17 @@
 # limitations under the License.
 #
 
-FROM centos:7
+# This Dockerfile may be used during development. It adds copies built binaries from distribution/target folder
```

Comment:
adds or copies?

Reply:
Thanks, adds only should be there, fixed.